README.md

```diff
@@ -10,6 +10,7 @@ The Docker extension makes it easy to build, manage and deploy containerized app
 * Command Palette (`F1`) integration for the most common Docker commands (for example `docker build`, `docker push`, etc.)
 * Explorer integration for managing Images, running Containers, and Docker Hub registries
 * Deploy images from Docker Hub and Azure Container Registries directly to Azure App Service
+* Debug .NET Core applications running in Linux Docker containers
 * [Working with docker](https://code.visualstudio.com/docs/azure/docker) will walk you through many of the features of this extension
 
```

Comment:
Seems like the ASP.NET debugging should appear lower in the README after the more general functionality.

Reply:
Moved to below the private registries (preview) section.